### PR TITLE
PROUD group contribution from 2021 WTD style guide group, thank you all: command line, release notes, edits, fixes, and other sections

### PR DESCRIPTION
**Pull request by barriebyron**
@theomelo and @barriebyron will work here, all contributions and partners welcome 

We learned git, we learned fuzzy search with `t`, we learned `git add -p`, we learned soft wrap, we learned group fun, and how awesome it is together.

### CODE REVIEW

**Comment by barriebyron** [comment]
companion to https://github.com/writethedocs/www/pull/1504

**Comment by theomelo** [comment]
companion to https://github.com/writethedocs/www/pull/1506

**Comment by barriebyron** [comment]
@Bradamant3 please help us make it happen!

File: docs/guide/writing/style-guides.md [hunk 1]
```diff
@@ -154,41 +112,85 @@ Effective FAQ pages accomplish the following:
...
+- [App Release Notes Are Getting Stupid - TechCrunch](https://techcrunch.com/2015/09/04/app-release-notes-are-getting-stupid/)
 
 Related talks:
 
 - [Learning to love release notes](http://www.writethedocs.org/videos/prague/2018/learning-to-love-release-notes-anne-edwards/) at Write the Docs Prague 2018
+- [Alexander Koren - Rethinking Release Notes](https://www.youtube.com/watch?v=SWduFnDPjYg) at Write the Docs Australia 2019
+
+### Error messages
+
+Mistakes in software are unavoidable. Users might click a link that turns out to be broken or they might perform an action only to be met with an error. When a user encounters a problem, it's important to write error messages in a way that doesn't alienate the user.
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]: I'm not sure I would personally call that a 'mistake' though. The click is intentional, even if the link is broken. How about this? 

```suggestion
Errors in software are unavoidable. Users might click a link that turns out to be broken or they might perform an action only to be met with an unexpected outcome. When a user encounters a problem, it's important to write error messages in a way that doesn't alienate the user.
```

**Review by plaindocs — CHANGES_REQUESTED** [review]
This is great, I've made a suggested change, and will tackle the merge issue with #1504 and or #1508 when we get there.

**Comment by plaindocs** [comment]
Oh, and there is an introduced broken link due to a heading rename that could do with fixing

```
- docs/_build/html/blog/newsletter-november-2020/index.html
  *  linking to internal hash #reducing-bias-in-your-writing that does not exist (line 106)
     <a class="reference external" href="https://www.writethedocs.org/guide/writing/style-guides/#reducing-bias-in-your-writing">lists of resources for inclusive writing and accessibility tooling</a>
```

**Comment by barriebyron** [comment]
hi @plaindocs looks like the newsletter link is already updated for the new bias content section championed by @Bradamant3 

I alphabetized our lists and made a few more edits in the PR. lmk what other changes might be required. thank you!

**Review by plaindocs — APPROVED** [review]

**Comment by plaindocs** [comment]
@barriebyron looks good to me, press the green button once that double hyphen commit goes though (assuming CI passes).

File: docs/guide/writing/style-guides.md [hunk 2]
```diff
@@ -31,7 +30,7 @@ Classic style guides for software documentation include:
 Other useful resources:
 
 - [18F US Government Content Guide](https://content-guide.18f.gov)
-- [DigitalOcean Documentation Style Guide](https://docs.digitalocean.com/style/
+- [DigitalOcean Documentation Style Guide](<https://docs.digitalocean.com/style/>
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]: I think you want 

```
[DigitalOcean Documentation Style Guide](https://docs.digitalocean.com/style/)
```

File: docs/guide/writing/style-guides.md [hunk 3]
```diff
@@ -152,20 +151,21 @@ Consider the following when creating an entry for your release notes:
...
+- [Transforming Your Release Notes into Product Announcements -- Parlor](https://www.parlor.io/blog/transforming-your-release-notes-into-product-announcements/)
+- [The art of writing great release notes -- UX Collective](https://uxdesign.cc/the-art-of-writing-great-release-notes-6607e22efae1)
+- [The art of writing Release Notes A guide, examples & template -- Slite](https://slite.com/learn/release-notes)
+- [The best and the worst app release notes that will inspire you -- announcekit](https://announcekit.app/blog/the-best-and-the-worst-app-release-notes-that-will-inspire-you)
 
 Examples of great release notes:
 
 - [DataStax DSE](https://docs.datastax.com/en/dse/6.8/dse-admin/datastax_enterprise/releaseNotes/releaseNotes.html)
-- [Digital Ocean](https://docs.digitalocean.com/release-notes/
+- [Digital Ocean](<https://docs.digitalocean.com/release-notes/>
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]: same here

**Comment by plaindocs** [comment]
Preview here https://writethedocs-www--1501.org.readthedocs.build/guide/writing/style-guides/#style-guide-resources

**Comment by plaindocs** [comment]
:tada: !

**Comment by barriebyron** [comment]
thank you everyone for contributing and collaborating, our updated Style Guide page is live https://www.writethedocs.org/guide/writing/style-guides/

**Pull request merged**